03-setup-ingress-controller/guides/wildcard_certificates.md

```diff
@@ -0,0 +1,682 @@
+
+Typical `Certificate` CRD for wildcard certificates looks like below (explanations inline):
+
+```yaml
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: mydomain.org
+  # Cert-Manager will put the resulting Secret in the same Kubernetes namespace as the Certificate.
+  namespace: dev
+spec:
+  # Secret name to create, where the private key and certificate should be stored.
+  secretName: mydomain.org
+  # What Issuer to use for getting the certificate.
+  issuerRef:
+    name: letsencrypt-dev-wcard
+    kind: Issuer
+    group: cert-manager.io
+  # Common name to be used on the Certificate.
+  commonName: "*.mydomain.org"
+  # List of DNS subjectAltNames to be set on the Certificate.
+  dnsNames:
+    - "mydomain.org"
+    - "*.mydomain.org"
+```
+
+Next, open and inspect the `Certificate` manifest file provided in the `Starter Kit` repository, using an editor of your choice (preferably with `YAML` lint support). For example, you can use [VS Code](https://code.visualstudio.com). Please pick only one option, depending on the installed `Ingress Controller`:
+
+- `Ambassador Edge Stack` ingress:
+
+  ```shell
+  code 03-setup-ingress-controller/assets/manifests/cert-manager-ambassador-wcard-certificate.yaml
+  ```
+
+- `Nginx` ingress:
+
+  ```shell
+  code 03-setup-ingress-controller/assets/manifests/cert-manager-nginx-wcard-certificate.yaml
+  ```
+
+**Notes:**
+
+- Explanation for each important field of the `Certificate` CRD, can be found inside the [cert-manager-ambassador-wcard-certificate.yaml](../assets/manifests/cert-manager-ambassador-wcard-certificate.yaml) (or [cert-manager-nginx-wcard-certificate.yaml](../assets/manifests/cert-manager-nginx-wcard-certificate.yaml)) file.
+- The example provided in this tutorial is using `starter-kit.online` domain name (and naming convention). Please make sure to use your own domain name and naming convention.
+
+Next, create the `Certificate` resource in your `DOKS` cluster (please pick only one option, depending on the installed `Ingress Controller`):
+
+- `Ambassador Edge Stack` ingress:
+
+  ```shell
+  kubectl apply -f 03-setup-ingress-controller/assets/manifests/cert-manager-ambassador-wcard-certificate.yaml
+  ```
+
+- `Nginx` ingress:
+
+  ```shell
+  kubectl apply -f 03-setup-ingress-controller/assets/manifests/cert-manager-nginx-wcard-certificate.yaml
+  ```
+
+Verify certificate status (please pick only one option, depending on the installed `Ingress Controller`):
+
+- `Ambassador Edge Stack` ingress:
+
+  ```shell
+  kubectl get certificate starter-kit.online -n ambassador
+  ```
+
+- `Nginx` ingress:
+
+  ```shell
+  kubectl get certificate starter-kit.online -n ingress-nginx
```

Comment:
The command should be:
kubectl get certificate starter-kit.online -n backend

Reply:
Should be fixed in the latest commit. Thanks.